app/models/soft_delete.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth.models import UserManager
+from django.db import models
+from django.utils import timezone
+
+
+class SoftDeleteManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(deleted_at__isnull=True)
+
+
+class SoftDeleteModel(models.Model):
+
+    deleted_at = models.DateTimeField(null=True, blank=True)
```

Comment:
Can you also add `is_deleted = models.BooleanField`?

Reply:
Also add an index for `is_deleted` field.

Reply:
ok